server/src/test/java/org/elasticsearch/search/fieldcaps/FieldCapabilitiesIT.java

```diff
@@ -148,4 +148,11 @@ public void testFieldAliasFilteringWithWildcard() {
         assertEquals(1, response.get().size());
         assertTrue(response.get().containsKey("distance"));
     }
+
+    public void testEmptyIndexPattern() {
+        FieldCapabilitiesResponse response = client().prepareFieldCaps("empty_index_pattern*")
+            .setFields("*")
+            .execute().actionGet();
+        assertEquals(0, response.get().size());
```

Comment:
This could be done from a unit test as well right? Create the response using the default constructor and call writeTo on it.